File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/controller/dto/SvarAnmodningUnntakDto.java
package no.nav.melosys.eessi.controller.dto;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import lombok.AllArgsConstructor;
import lombok.Data;
import no.nav.melosys.eessi.models.sed.medlemskap.impl.SvarAnmodningUnntakBeslutning;

@Data
@AllArgsConstructor
@JsonIgnoreProperties(ignoreUnknown = true)
public class SvarAnmodningUnntakDto {

    private SvarAnmodningUnntakBeslutning beslutning;
    private String begrunnelse;
    private Periode delvisInnvilgetPeriode;
}

File: melosys-eessi-app/src/main/java/no/nav/melosys/eessi/controller/dto/SvarAnmodningUnntakDto.kt
package no.nav.melosys.eessi.controller.dto

import com.fasterxml.jackson.annotation.JsonIgnoreProperties
import no.nav.melosys.eessi.models.sed.medlemskap.impl.SvarAnmodningUnntakBeslutning

@JsonIgnoreProperties(ignoreUnknown = true)
data class SvarAnmodningUnntakDto(
    var beslutning: SvarAnmodningUnntakBeslutning? = null,
    var begrunnelse: String? = null,
    var delvisInnvilgetPeriode: Periode? = null
)
